src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1056,10 +1056,6 @@ private function addValidationSection(ArrayNodeDefinition $rootNode, callable $e
                         ->arrayNode('not_compromised_password')
                             ->canBeDisabled()
                             ->children()
-                                ->booleanNode('enabled')
-                                    ->defaultTrue()
-                                    ->info('When disabled, compromised passwords will be accepted as valid.')
```

Comment:
Ne feature proposal: `canBeDisabled` could accept a parameter for the info.

Reply:
I think you're right and I should work on that instead, seems a better option to not loose info, even if it seems self-explanatory in this case.